README.md

```diff
@@ -21,3 +23,37 @@ params = {
 }
 response = requests.post(url, params = params)
 ```
+
+### Verify OTP
+Use the below endpoint to verify OTP code sent to a mobile number.
+```
+https://ym6gqg7jui.execute-api.ap-south-1.amazonaws.com/dev/verifyotp
+```
+Required query parameters: `phone`, `code`
+
+Optional query parameters: `msg`, `otpCodeLength`, `optCodeType`
+
+#### Example
+```py
+import requests
+url = "https://ym6gqg7jui.execute-api.ap-south-1.amazonaws.com/dev/verifyotp"
+params = {
+  "phone": "9999999999",
+  "code":"1234"
+}
+response = requests.post(url, params = params)
+```
+
+### Query Parameters
+`phone` : The number must be in pure numeric format with no special characters. <br/>
+
+`msg` : The message that needs to be sent. It can contain alphanumeric & special characters. The message must contain `%code%`.
```

Comment:
This is currently not configurable, right? Or did you change this? If not configurable, mention it

Reply:
It is configurable!